hnn_core/parallel_backends.py

```diff
@@ -0,0 +1,265 @@
+"""Parallel backends"""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mainakjas@gmail.com>
+
+from warnings import warn
+
+_backend = None
+_loaded_dll = None
+
+
+class Joblib_backend(object):
```

Comment:
I think this should be a private class and class names are always in CamelCase

Reply:
Renamed: JoblibBackend